src/redclaw/data.clj

```diff
@@ -41,11 +47,66 @@
 (defn read-all [rs]
   (.readAll rs))
 
+(defn remove [rs v]
+  (.remove rs v))
+
+(defn contains? [rs s]
+  (.contains rs s))
+
+(defn contains-all? [rs xs]
+  (.containsAll rs xs))
+
+(defn contains-some? [rs xs]
+  (->> (some xs (read-all rs))
+       boolean))
+
+;; -----------------------------------------------------------------
+;; functions for map actions
+;; -----------------------------------------------------------------
+
 (defn put [rm k v]
   (.put rm k v))
 
 (defn put-all [rm m]
   (.putAll rm m))
 
-(defn get-all [rm]
-  (.getAll rm))
+(defn get [rm k]
+  (.get rm k))
+
+(defn get-all
+  ([rm] (get-all rm (.keySet rm)))
+  ([rm xs] (.getAll rm xs)))
+
+(defn get-keys [rm]
+  (.keySet rm))
+
+(defn has-key? [rm k]
+  (.containsKey rm k))
```

Comment:
these common across all need to be kept one to one
the idea is to be consistent with the thing we wrap

the whole idea to wrap these one to one is so they become composable `IFn` (clojure function)
anything beyond that should be independently applied to a Reddison type
or later, maybe be a separate common tools set